api/lib/infrastructure/repositories/user-repository.js

```diff
@@ -227,11 +227,11 @@ module.exports = {
   updatePassword(id, hashedPassword) {
     return BookshelfUser
       .where({ id })
-      .save({ password: hashedPassword }, {
-        patch: true,
-        require: false
-      })
-      .then((bookshelfUser) => bookshelfUser.toDomainEntity());
+      .save({ password: hashedPassword }, { patch: true, method: 'update' })
+      .then((bookshelfUser) => bookshelfUser.toDomainEntity())
+      .catch(() => {
+        throw new UserNotFoundError();
+      });
```

Comment:
ici tout ce que l'on catch est une not found ? Je ne pense pas. Je serais plutôt d'avis de faire comme pour studentRepository.get
```
      .catch((err) => {
        if (err instanceof BookshelfUser.NotFoundError) {
          throw new NotFoundError(`User not found for ID ${userId}`);
        }
        throw err;
      });
```

Reply:
ok